src/easy/handler.rs

```diff
@@ -428,7 +428,7 @@ pub enum IpResolve {
 
 /// Possible values to pass to the `http_version` method.
 #[non_exhaustive]
-#[derive(Debug, Clone, Copy)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
 pub enum HttpVersion {
```

Comment:
Curious why simple enums have to be compared by `match`/`if let` and are not allowed to be used in simple `==` by deriving these quite-standard `Eq` traits. Intentional or oversight?